docs/documentation/configuration.mdx

```diff
@@ -14,6 +14,76 @@ SET paradedb.enable_custom_scan = true;
 Custom scans should only be disabled for debugging purposes. They must be enabled for scoring, highlighting,
 and various predicate pushdowns to work.
 
+## Indexing Resource Usage
```

Comment:
Would be awesome to link to this in the "Create an Index" docs

Reply:
can you link me to those so I know exactly where you mean?

Reply:
nvm.  I think I found 'em.